src/CodeStyle/Core/Analyzers/PublicAPI.Unshipped.txt

```diff
@@ -68,8 +67,6 @@ Microsoft.CodeAnalysis.Internal.Editing.DeclarationModifiers.WithIsVolatile(bool
 Microsoft.CodeAnalysis.Internal.Editing.DeclarationModifiers.WithIsWriteOnly(bool isWriteOnly) -> Microsoft.CodeAnalysis.Internal.Editing.DeclarationModifiers
 Microsoft.CodeAnalysis.Internal.Editing.DeclarationModifiers.WithPartial(bool isPartial) -> Microsoft.CodeAnalysis.Internal.Editing.DeclarationModifiers
 Microsoft.CodeAnalysis.Internal.Editing.DeclarationModifiers.WithWithEvents(bool withEvents) -> Microsoft.CodeAnalysis.Internal.Editing.DeclarationModifiers
-override Microsoft.CodeAnalysis.Internal.Editing.DeclarationModifiers.Equals(object obj) -> bool
```

Comment:
📝 This appears to be a bug in Public API Analyzers

Reply:
let me see about fixing this in public API.